src/code.rs

```diff
@@ -314,6 +314,12 @@ impl<'a> Struct<'a> {
                     StructType::Update => self.opts.get_update_str_type().as_str().to_string(),
                     StructType::Create => self.opts.get_create_str_type().as_str().to_string(),
                 }
+            } else if f.base_type == "Vec<u8>" {
+                f.base_type = match self.ty {
+                    StructType::Read => f.base_type,
+                    StructType::Update => self.opts.get_update_bytes_type().as_str().to_string(),
+                    StructType::Create => self.opts.get_create_bytes_type().as_str().to_string(),
+                }
```

Comment:
in this file the `lifetimes` also will need to be updated, at https://github.com/Wulf/dsync/blob/3f3d57c36647451403c907c6078764fa3894463d/src/code.rs#L293-L298